src/main.cpp

```diff
@@ -510,19 +536,19 @@ void tick_all_interfaces(const SysTick_s &current_system_tick)
             int(fsm.get_state()),
             buzzer.buzzer_is_on(),
             drivetrain.drivetrain_error_occured(),
-            torque_controller_mux.getTorqueLimit(),
+            torque_controller_mux.get_tc_mux_status().current_torque_limit_enum,
             ams_interface.get_filtered_min_cell_voltage(),
             a1.get().conversions[MCU15_GLV_SENSE_CHANNEL],
-            static_cast<int>(torque_controller_mux.activeController()->get_launch_state()),
+            static_cast<int>(torque_controller_mux.get_tc_mux_status().current_controller_mode_),
```

Comment:
if current_controller_mode_ is a private member variable how are we accessing it here

Reply:
`current_control_mode_` is just a member of a struct, not a private member. 

I am removing the trailing underscore as it is not a private member.

also the trailing underscore for private members is just a naming convention, not something that the compiler cares about.